js-intro.js

```diff
@@ -36,9 +60,17 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+removeVowel = (str) => {
+  let newArr1 = str.split("")
+  let newArr2 = newArr1.filter(value => value.toLowerCase() !== "a" && value.toLowerCase() !== "e" && value.toLowerCase() !== "i" && value.toLowerCase() !== "o" && value.toLowerCase() !== "u")
```

Comment:
You can actually add `.join("")` to the end of this line to save yourself time! You can save yourself a line of code by returning your filtered array outright, rather than creating a new variable in order to turn around and immediately return it. Filter is an accessor which doesn't permanently change to original array, so if you wanted to use those values later in your code you would need to save it to a new variable like you did. But for this purpose, you can save yourself some key strokes :)